components/cam/tools/mkatmsrffile/README

```diff
@@ -1,25 +1,21 @@
-Atmospheric drydeposition at the surface depends on certain surface
-properties including soil and land use properties.  In most cases
-these calculations can be handled in the land model and passed to he
-atmosphere through the coupler.  This is the default namelist setting
-drydep_method='xactive_lnd'.  However with modal areosols this method
-is not adequate and we must recalculate these fields in the atmosphere
-(see subroutine interp_map in mo_drydep.F90).  For unstructured grids
-it was determined to create this offline interpolation tool rather
-than generalize the subroutine interp_map.  
-
-1. Locate or create scrip mapping files for the 1x1 degree input grids and
-   the desired model output grid.
+Atmospheric drydeposition at the surface depends on certain surface properties including soil and land use properties.  In most cases, these calculations can be handled in the land model and passed to the atmosphere through the coupler.  This method is the default namelist setting drydep_method='xactive_lnd'. 
+
+However with modal areosols, this method is not adequate, and we must recalculate these fields in the atmosphere (see subroutine interp_map in mo_drydep.F90).  
+
+For unstructured grids, it was determined to create this offline interpolation tool rather than generalize the subroutine interp_map.  
+
+
+1. Locate or create scrip mapping files for the 1x1 degree input grids and the desired model output grid.
 
 2. Run the script scrip_makemaps.csh 
```

Comment:
We should remove this mention of scrip_makemaps.csh, and replace with your additional text below about ESMF_RegridWeightGen.